resources/assets/components/InboxItem/InboxTile.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+class InboxTile extends React.Component {
+  displayImage(photo_url) {
```

Comment:
@sbsmith86 making this a helper is a great idea! are you thinking it would have the same arguments and return the same things that it does here? also the default image is hardcoded for now, but we'll want to pick a different one eventually